app/controllers/application_controller_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "rails_helper"
```

Comment:
Wrong place for this file: `app/controllers/...`

Reply:
d'oh